src/js/components/artifacts/releaserepository.js

```diff
@@ -188,8 +195,9 @@ export class ReleaseRepository extends React.Component {
       );
     }
 
+    const noArtifactsClass = release ? '' : 'muted';
     return (
-      <div className="relative release-repo margin-left" style={{ width: '100%' }}>
+      <div className={`relative release-repo margin-left${wasSelectedRecently ? ' blurred' : ''}`} style={{ width: '100%' }}>
```

Comment:
My only problem with this is that it's not consistent with the rest of the UX – it's the only time this blurring effect is used, so it feels a bit unusual to me. Maybe in the short term we can use the standard loading spinner we use elsewhere, and in the medium-term we can find a nice smooth 'loading' effect to use throughout the UI?

Reply:
agreed, aligning it is better for now and it should now be based on the loader, I thought the blur might be a bit less "intrusive" as the data already exists in the store